src/components/About.js

```diff
@@ -19,13 +19,19 @@ export default class About extends Component {
         </p>
         <figure className="profile">
           <a href="https://github.com/swashcap/">
-            <img src="https://avatars0.githubusercontent.com/u/1858316" />
+            <img
+              alt="swashcap avatar"
```

Comment:
_Super_ gross diff. Big change: adding `alt` attributes to images to satisfy react-script's linter's JSX accessibility rules.